src/styles/constants/_layout.scss

```diff
@@ -2,7 +2,7 @@
 $modal-dialog-z-index: 160 !default;
 $menu-z-index: 170 !default;
 $submenu-z-index: 171 !default;
-$notifications-wrapper-z-index: 180 !default;
+$notifications-wrapper-z-index: 191 !default;
```

Comment:
minor: Seems like these variables are sorted by z-index value, when you get a chance, could we fix the ordering? Similar comment on the JS equivalent below.

Also having it be 1 more than the overlay/tooltip doesn't leave much room for anything else in between. It seems like we've used increments of 10 for the other step ups

Reply:
Sure! I'll update this